vnext/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -449,8 +449,8 @@ folly::dynamic ScrollViewManager::GetExportedCustomDirectEventTypeConstants() co
 XamlView ScrollViewManager::CreateViewCore(int64_t tag) {
   const auto scrollViewer = winrt::ScrollViewer{};
 
-  scrollViewer.HorizontalScrollBarVisibility(winrt::ScrollBarVisibility::Visible);
-  scrollViewer.VerticalScrollBarVisibility(winrt::ScrollBarVisibility::Visible);
+  scrollViewer.HorizontalScrollBarVisibility(winrt::ScrollBarVisibility::Auto);
```

Comment:
Can't you just remove the set altogether?  Seems like we should just use the default value from the XAML theme.

Reply:
@acoates-ms I tested this, and when those two lines of code are removed, I still see the incorrect behavior, which is that the scrollbars display even though the content I am displaying is not large enough to require a scrollbar.